src/cmd/package.go

```diff
@@ -46,8 +45,8 @@ var packageCreateCmd = &cobra.Command{
 			baseDir = args[0]
 		}
 
-		if zarfImageCache != config.ZarfDefaultImageCachePath && cachePathClean(zarfImageCache) {
-			config.SetImageCachePath(zarfImageCache)
+		if isCachePathClean(config.CreateOptions.CachePath) {
+			config.SetCachePath(config.CreateOptions.CachePath)
```

Comment:
We also try to reserve setters for non-public variables.  If the variable is exposed publicly it will likely be read/written publicly in some places (i.e. Cobra flags) so for consistency we only access it via the public read/write as any future getter/setter would not be used every time otherwise.

Reply:
refactored